<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="119" started-at="2019-09-14T10:33:07Z" finished-at="2019-09-14T10:33:07Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="119" started-at="2019-09-14T10:33:07Z" finished-at="2019-09-14T10:33:07Z">
      <class name="com.training.regression.tests.TrainerDBtest">
        <test-method status="PASS" signature="setUpBeforeClass()[pri:0, instance:com.training.regression.tests.TrainerDBtest@523884b2]" name="setUpBeforeClass" is-config="true" duration-ms="36" started-at="2019-09-14T16:03:07Z" finished-at="2019-09-14T16:03:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeClass -->
        <test-method status="SKIP" signature="loginDBTest(java.lang.String, java.lang.String)[pri:0, instance:com.training.regression.tests.TrainerDBtest@523884b2]" name="loginDBTest" duration-ms="0" started-at="2019-09-14T16:03:07Z" data-provider="db-inputs" finished-at="2019-09-14T16:03:07Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[Can't invoke method public java.lang.Object[][] com.training.dataproviders.LoginDataProviders.getDBData() on this instance of class com.training.regression.tests.TrainerDBtest due to class mismatch]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: Can't invoke method public java.lang.Object[][] com.training.dataproviders.LoginDataProviders.getDBData() on this instance of class com.training.regression.tests.TrainerDBtest due to class mismatch
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:89)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:51)
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:41)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:110)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1240)
	at org.testng.internal.Invoker.createParameters(Invoker.java:980)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1070)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDBTest -->
      </class> <!-- com.training.regression.tests.TrainerDBtest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
